### PR TITLE
[Instrumentation.Cassandra] Release 1.0.0-beta.1

### PR DESCRIPTION
**Pull request by xsoheilalizadeh**
Releasing 1.0.0

## Changes

Please provide a brief description of the changes here.

For significant contributions please make sure you have completed the following items:

* [x] Appropriate `CHANGELOG.md` updated for non-trivial changes
* [x] Design discussion issue #
* [x] Changes in public API reviewed

### CODE REVIEW

File: src/OpenTelemetry.Instrumentation.Cassandra/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Changelog
 
-## Unreleased
```
Comment (Kielek) [inline src/OpenTelemetry.Instrumentation.Cassandra/CHANGELOG.md:3]: Please keep Unreleased section.

File: src/OpenTelemetry.Instrumentation.Cassandra/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
+## 1.0.0
```
Comment (Kielek) [inline src/OpenTelemetry.Instrumentation.Cassandra/CHANGELOG.md:3]: Consider to release as 1.0.0-alpha.1/1.0.0-beta.1 or even (0.1.0-beta.1). to signalize that it is brand new package and it was not tested in the wild.

I think that we should not release this as a stable before https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/database-metrics.md reach stability (and this package will implement this metrics).

Comment (xsoheilalizadeh) [reply src/OpenTelemetry.Instrumentation.Cassandra/CHANGELOG.md]: Done

File: src/OpenTelemetry.Instrumentation.Cassandra/CHANGELOG.md [hunk 3]
```diff
@@ -2,4 +2,8 @@
 
 ## Unreleased
 
-Initial release.
+## 1.0.0-beta.1
+
+Released 2023-March-29
```
Comment (utpilla) [inline src/OpenTelemetry.Instrumentation.Cassandra/CHANGELOG.md:7]:
```suggestion
Released 2023-Mar-29
```

**Comment by Kielek** [comment]
I will handle release process tomorrow morning.

File: src/OpenTelemetry.Instrumentation.Cassandra/CHANGELOG.md [hunk 4]
```diff
@@ -2,4 +2,8 @@
 
 ## Unreleased
 
-Initial release.
+## 1.0.0-beta.1
+
+Released 2023-Mar-29
```
Comment (Kielek) [inline src/OpenTelemetry.Instrumentation.Cassandra/CHANGELOG.md:7]:
```suggestion
Released 2023-Mar-30
```

**Review by Kielek — APPROVED** [review]

**Comment by Kielek** [comment]
@xsoheilalizadeh released: https://www.nuget.org/packages/OpenTelemetry.Instrumentation.Cassandra/1.0.0-beta.1

**Comment by xsoheilalizadeh** [comment]
Thanks @Kielek

**Pull request merged**